x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/repositories/GetCcrRestoreFileChunkAction.java

```diff
@@ -72,27 +73,27 @@ protected void doExecute(Task task, GetCcrRestoreFileChunkRequest request,
         }
     }
 
-    public static class GetCcrRestoreFileChunkResponse extends ActionResponse {
+    public static class GetCcrRestoreFileChunkResponse extends ActionResponse implements RefCounted {
```

Comment:
I think this means the comment above ("This is currently safe to do ...") isn't needed any more. I mean it's still safe to do, but not for the reason it gives any more.

Reply:
++ Removed the comment